### ISSUE TITLE
Problem

### ISSUE THREAD

**Issue by AnZRPY** [BedrockLauncher/BedrockLauncher#142]
Everytime i launch it, its still in trial mode idk how to fix it pls help

**Comment by Rayth** [comment]
Do you actually own minecraft? You must own minecraft in order to use the launcher, this is not a pirating program.

Please see https://bedrocklauncher.github.io/faq

**Comment by AnZRPY** [comment]
i own the trial mc, do i need to buy the paid version?

**Comment by Rayth** [comment]
Yes you need the paid version, otherwise your account doesn't have permissions to download the game from the store

**Comment by AnZRPY** [comment]
Aight thx for answering :>
